src/main.rs

```diff
@@ -441,7 +441,7 @@ fn run() -> Result<(), Error> {
             release,
             strip,
         } => {
-            let venv_dir = match env::var_os("VIRTUAL_ENV") {
+            let venv_dir = match env::var_os("VIRTUAL_ENV").xor(env::var_os("CONDA_PREFIX")) {
                 Some(dir) => PathBuf::from(dir),
                 None => {
                     bail!("You need be inside a virtualenv to use develop (VIRTUAL_ENV isn't set)")
```

Comment:
Please change this error message to also include conda.

Reply:
Thanks for pointing this out. Comment addressed.